### PR TITLE
[sub-plugin] Add function to stop model training

### PR DESCRIPTION
**Pull request by songgot**
[sub-plugin] Add function to stop model training
    
nnstreamer tensor_trainer call this function to stop model training

 **Self evaluation:**
 1. Build test:   [X]Passed [ ]Failed [ ]Skipped
 2. Run test:     [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc [hunk 1]
```diff
@@ -578,19 +602,30 @@ void NNTrainer::NNTrainerTrain::createModel() {
...
     model->initialize();
   } catch (const std::exception &e) {
     ml_loge("Error %s, %s", typeid(e).name(), e.what());
     return;
   }
+
+  try {
+    if (!model_load_path.empty()) {
+      ml_loge("load path : %s", model_load_path.c_str());
+      model->load(model_load_path, ml::train::ModelFormat::MODEL_FORMAT_BIN);
```
Comment (DonghakPark) [inline nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc:622]: Does the intended flow : load configure(ini file) first and load weight if there is bin file ?

**Review by DonghakPark — APPROVED** [review]

File: nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc (hunk 1 above)
Comment (songgot) [reply nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc:622]: Yes.

**Review by djeong20 — APPROVED** [review]
LGTM

**Review by baek2sm — APPROVED** [review]
LGTM

**Review by again4you — APPROVED** [review]
LGTM!

**Review by gichan-jang — APPROVED** [review]

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**